packages/rx_bloc_cli/lib/src/models/generator_arguments.dart

```diff
@@ -132,5 +132,9 @@ class GeneratorArguments
   @override
   bool get cicdCodemagicEnabled => _featureConfiguration.cicdCodemagicEnabled;
 
+  /// CI/CD Gitlab
+  @override
+  bool get cicdGitlabEnabled => _featureConfiguration.cicdGitlabEnabled;
+
```

Comment:
The PR `fix/rx_bloc-score` includes changes to increase coverage of the rx_bloc_cli package. Since I see that this will be released as the `5.1.0` (meaning after the rx_bloc ecosystem gets an update), you can include those changes to your branch after it is merged And add your code-specific tests.

Reply:
I have updated with the latest changes, but I am not sure what kind of tests I can write for pipeline. Have you ever written such?

Reply:
I have updated tests, regarding your explanation, thanks for it! Coverage now looks suffitiont